libs/community/langchain_community/retrievers/tavily_search_api.py

```diff
@@ -31,14 +31,18 @@ def _get_relevant_documents(
         self, query: str, *, run_manager: CallbackManagerForRetrieverRun
     ) -> List[Document]:
         try:
-            from tavily import Client
+            try:
+                from tavily import TavilyClient
+            except TypeError:
+                # Older of tavily used Client
+                from tavily import Client as TavilyClient
```

Comment:
@2020uce0047 added path for compatibility with older clients. Could you confirm that this makes sense?